main.js

```diff
@@ -236,5 +224,5 @@ module.exports.static = express.static;
 module.exports.services = metrics.services;
 module.exports.metrics = metrics;
 module.exports.flags = flags;
-// TODO: for backwards compatiability, but modules/apps can use next-logger directly, as of v4.0.0
-module.exports.logger = nextLogger;
+// TODO: for backwards compatiability, but modules/apps can use n-logger directly, as of v4.0.0
+module.exports.logger = nLogger;
```

Comment:
n-logger is only instantiated once anyway (https://github.com/Financial-Times/n-logger/blob/master/src/main.js#L80-L89), initially it was in here so it could get the name of the app that was running, to construct the splunk url. now that's a config-var, there's no reason to use this. however, there might still be apps using it this way. 

in summary, this should be removed, just need to do a sweep of the apps to make sure nothing's using it